### DISCUSSION TITLE
How do I remove items from the toolbar?

### DISCUSSION THREAD

**Discussion by duneboy84** [losnoco/Cog#342] [Q&A] [answered]
I may be doing something wrong, but how do I remove items from the toolbar? I can add them, rearrange them but cannot remove them. I drag the icons from the tool bar using the command key but nothing happens.

**Comment by kode54** [comment]
Apparently macOS older than Ventura has this bug with anything built with Xcode 14. I don't know what I can do about it, other than let CI build the app with Xcode 13.0 or older.

**Reply by duneboy84** [reply]
That explains it then since I am running Monterey. I won't be going to Ventura anytime soon since my hardware is old. Thank you for the reply.

**Comment by kode54** [comment, marked as answer]
I just pushed a build made with Xcode 13.3.1, this may fix it.

**Reply by duneboy84** [reply]
The new build allows me to remove menu items. Thank you.
